at_app_flutter/lib/src/templates/app/main.dart

```diff
@@ -58,6 +60,9 @@ class _MyAppState extends State<MyApp> {
                   appAPIKey: AtEnv.appApiKey,

```

Comment:
domain is no more needed. Staging and production domains are added in the widget itself. Check it once and make changes if you think it is necessary.

Reply:
I saw, what I am doing is overriding the environment in case they want to test on prod but they don't have an api key yet.